shared/src/commonMain/kotlin/io/customer/shared/sdk/config/BackgroundQueueConfig.kt

```diff
@@ -0,0 +1,61 @@
+package io.customer.shared.sdk.config
+
+/**
+ * Configurations to hold background queue settings.
+ */
+class BackgroundQueueConfig private constructor(
+    val batchDelayMinTasks: Int,
+    val batchDelayMaxDelayInSeconds: Int,
+    val batchTasksMax: Int,
```

Comment:
I think these names can be improved, but I'm little stuck here, so open for suggestions.